.pylintrc

```diff
@@ -4,7 +4,12 @@ disable=
     C0115, # missing-class-docstring
     C0116, # missing-function-docstring
     R0903, # Too few public methods (0/2) (too-few-public-methods)
+    W0105, # String statement has no effect
     W0221, # Parameters differ from overridden
+    W0237, # Parameter has been renamed to.. in overridden
```

Comment:
Why did you need all these statements?

Reply:
i disabled this because in the generic repository i created methods with parameters like:
get_by_id(obj_id):
.......
and then when i inherit the method from generic i overridden obj_id by produt_id, chef_id, order_id etc, depending the repository.

Reply:
What about the other ones?

Reply:
done.